src/index.html

```diff
@@ -11,7 +11,156 @@
+              min="1"
+              max="100"
+              value="12"
+              class="age_input input"
+            >
+          </label><br>
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              class="date_input input"
+            >
+          </label><br>
+          <label>
+            I accept the term of agreement
+            <input
+              type="checkbox"
+              required
+            >
+          </label><br>
+        </fieldset>
+      </label>
+
+      <!-- Registration -->
+      <label>
+        <fieldset class="reg_fieldset fieldset">
+          <legend>Registration</legend>
+          <label>
+            E-mail:
+            <input
+              type="email"
+              placeholder="email@example.com"
+              class="email_input input"
+            >
+          </label><br>
+          <label>
+            Password:
+            <input
+              type="password"
+              minlength="7"
+              maxlength="20"
+            >
+          </label><br>
+        </fieldset>
+      </label>
+
+      <!-- Facts group -->
+      <label>
+        <fieldset class="facts_fieldset fieldset">
+          <legend>An interesting fact about you!</legend>
+          <div class="radio input">
+             Do you love cats?
+            <label>
+              <input
+                name="cats"
+                type="radio"
+              >Yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="cats"
+              >
+              No
+            </label>
+          </div>
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              class="color_input input"
+            >
+          </label><br>
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              step="0.1"
+              class="time_input input"
+            >
+          </label>
+          <div class="select input">
+            <label>
+              What are your favorite brand of cars?
+              <select multiple>
+                <option value="bmw">BMW</option>
+                <option value="audi">Audi</option>
+                <option value="lada">Lada</option>
+                <option></option>
+              </select>
+            </label>
+          </div>
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              value="0"
+            >
+          </label>
+        </fieldset>
+      </label>
+
+      <!-- Additional info group -->
+      <label>
+        <fieldset class="info_fieldset fieldset">
+          <legend>Additional info</legend>
+          <div class="coms input">
+            <label>
+              Comments:
+              <textarea cols="16" rows="2">
+              </textarea>
```

Comment:
```suggestion
              <textarea cols="16" rows="2"></textarea>
```
textarea має особливість - якщо між відкриваючим і закриваючим тегами є пробіл або перенос - відпрацьовує некоректно, а саме курсор ставиться не на початок вводу